api/src/opentrons/protocol_api_experimental/instrument_context.py

```diff
@@ -107,14 +113,39 @@ def pick_up_tip(
             presses: Optional[int] = None,
             increment: Optional[float] = None) -> InstrumentContext:
         # noqa: D102
-        raise NotImplementedError()
+        # TODO(al, 2021-04-12): What about presses and increment? They are not
+        #  supported by PE command. They are also not supported by PD protocols
+        #  either.
+        if isinstance(location, Well):
+            self._client.pick_up_tip(
+                pipette_id=self._pipette_id,
+                labware_id=location.parent.id,
+                well_name=location.well_name
+            )
+        else:
+            # TODO(al, 2021-04-12): Support for picking up next tip in a labware
+            #  and in tipracks associated with a pipette
+            raise NotImplementedError()
+
+        return self
 
     def drop_tip(
             self,
             location: Union[types.Location, Well] = None,
             home_after: bool = True) -> InstrumentContext:
         # noqa: D102
-        raise NotImplementedError()
+        # TODO(al, 2021-04-12): What about home_after?
```

Comment:
```suggestion
        if not home_after:
            raise NotImplementedError()
```

Same as comment above.